Azure-Samples/msdocs-flask-postgresql-sample-app
# Port Flask sample to use keyword/value style connection string

## Purpose

This PR updates the code to use the environment variable currently created by the WebApp+Database portal template. As discussed, this PR changes only the production environment configuration, keeping local development the same.

## Does this introduce a breaking change?


```
[X] Yes (as in, previous code is breaking!)
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Follow tutorial steps for Github deployment
* Use 'azd up'
* Check you can add restaurants to both.
